pkg/scheduler/scheduler.go

```diff
@@ -169,20 +168,17 @@ func (s *Scheduler) RegisterFromNodeAnnotations() {
 			nodeNames = append(nodeNames, val.Name)
 			for devhandsk, devInstance := range device.GetDevices() {
 				health, needUpdate := devInstance.CheckHealth(devhandsk, val)
+				klog.V(5).InfoS("node ", val.Name, "device", devhandsk, "health", health, "needUpdate", needUpdate)
 				if !health {
-					_, ok := s.nodes[val.Name]
+					err := devInstance.NodeCleanUp(val.Name)
+					if err != nil {
+						klog.Errorln("node cleanup failed", err.Error())
```

Comment:
Yes。 Because the device-plugin is not healthy at this time. Therefore, it is necessary to remove the information stored in memory.

Reply:
so we should add `continue` ?

Reply:
> so we should add continue ?

No。 We need to continue executing downwards. Because the device-plugin is not healthy at this time. Therefore, it is necessary to remove the information stored in memory.